requirements_common.txt

```
@@ -10,7 +10,10 @@ loguru~=0.5
 joblib~=1.0
 matplotlib>=3.3.0
 nibabel~=3.2
-onnxruntime==1.7.0
+# https://github.com/microsoft/onnxruntime/issues/11680#issuecomment-1141228127
+--extra-index-url https://test.pypi.org/simple
+ort-nightly;python_version>="3.10"
```

This should be replaceable with 

```suggestion
onnxruntime>=1.7.0
```

as of mid-July: https://github.com/ivadomed/ivadomed/issues/1136#issuecomment-1141382059